internal/tool/generate/generator_test.go

```diff
@@ -135,7 +136,12 @@ func runGenerator(t *testing.T, directory, filename, contents string, subdirs []
 	if err := tidy.Run(); err != nil {
 		t.Fatalf("go mod tidy: %v", err)
 	}
-	gobuild := exec.Command("go", "build")
+
+	var buildTagsArg string
+	if len(buildTags) > 0 {
```

Comment:
I thought the earlier code always added "ignoreWeaveGen". So can we supply -tags unconditionally?

Reply:
Great observation. Fixed